<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyHomework</class>
 <widget class="QMainWindow" name="MyHomework">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QPushButton" name="prevday_push">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextday_push">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>60</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>15</y>
      <width>85</width>
      <height>15</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Date the day</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="homeworkTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>630</width>
      <height>350</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>251</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addles_push">
       <property name="text">
        <string>+ урок</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addhom_push">
       <property name="text">
        <string>+ дз</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delles_push">
       <property name="text">
        <string>- урок</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delhom_push">
       <property name="text">
        <string>-дз</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>импорт файла</string>
    </property>
    <addaction name="separator"/>
    <addaction name="choose_file"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="choose_file">
   <property name="text">
    <string>выбрать файл</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
